<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YtDownloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/youtube.svg</normaloff>../../../Downloads/youtube.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(225, 219, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: rgb(225, 219, 255);
}</string>
   </property>
   <widget class="QPushButton" name="get_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>9</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: transparent;
border: 1px solid rgb(144, 144, 144);
border-radius: 15px;
color: rgb(144, 144, 144);
}

QPushButton:hover {
background:  rgb(144, 144, 144);
color: white;
}</string>
    </property>
    <property name="text">
     <string>Get </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>55</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="url_link_field">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>10</y>
      <width>386</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background: rgb(193, 193, 193);
color: rgb(255, 255, 255);
border: 1px solid rgb(144, 144, 144);
border-radius: 15px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="url_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="download_btn">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>50</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: transparent;
border: 1px solid rgb(255, 35, 35);
border-radius: 13px;
color: rgb(255, 35, 35);
}

QPushButton:hover {
background:  rgb(255, 35, 35);
color: white;
}</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
